docassemble/ALToolbox/data/questions/al_income.yml

```diff
@@ -89,4 +89,602 @@ data: !!omap
+    datatype: currency
+    show if: x.is_hourly
+  - Amount you get paid: x.value
+    datatype: currency
+    show if:
+      variable: x.is_hourly
+      is: False
+  - What is the net income of the job?: x.net
+    datatype: currency
+---
+# UNIQUE ITEMS FOR ALJobList
+---
+id: how many regular jobs
+generic object: ALJobList
+question: |
+  How many jobs do you have?
+fields:
+  - no label: x.target_number
+    datatype: integer
+---
+id: name of one regular job in list
+generic object: ALJobList
+question: |
+  What is your job title?
+subquestion: |
+  % if len(x.complete_elements()) > 0:
+  You've already told us about ${ comma_and_list([job.source for job in x.complete_elements()]) }.
+  % endif
+fields:
+  - Job name: x[i].source
+---
+# SHARED BY ALAsset AND ALAssetList
+---
+id: asset info
+generic object: ALAsset
+question: |
+  Tell us about this asset
+fields:
+  - What type of asset is it?: x.source
+    code: |
+      asset_source_list
+    exclude: |
+      ['vehicle']
+  - How often do you get this income?: x.times_per_year
+    input type: radio
+    code: |
+      times_per_year_list
+    datatype: integer
+  - Interest or other income from the asset: x.value
+    datatype: currency
+  - Who owns this?: x.owner
+    required: False
+---
+# UNIQUE FOR ALAssetList
+---
+generic object: ALAssetList
+code: |
+  x[i].source
+  x[i].market_value
+  x[i].complete = True
+---
+id: number of assets
+generic object: ALAssetList
+question: |
+  How many assets do you have?
+fields:
+  - no label: x.target_number
+    datatype: integer
+---
+id: info for each asset
+generic object: ALAssetList
+question: |
+  About your ${ x[i].source } asset
+fields:
+  - Current market value: x[i].market_value
+    datatype: currency
+    required: False
+  - Balance (for a financial account, may be identical to market value): x[i].balance
+    datatype: currency
+    required: False
+  - How much does this asset earn you (in interest, dividends or other regular payments to you)?: x[i].value
+    datatype: currency
+    required: False
+  - How often do you get money for this asset?: x[i].times_per_year
+    input type: radio
+    code: |
+      times_per_year_list
+    datatype: integer
+    required: False
+---
+# UNIQUE FOR ALVehicle
+---
+generic object: ALVehicle
+code: |
+  # .source is automatically 'vehicle'
+  x.market_value
+  x.complete = True
+---
+comment: |
+  Notes:
+
+  A Vehicle is a child class of Asset, and so it has attributes like value
+  (representing income earned), times_per_year, balance, and market_value, but
+  for most vehicle owners only the `balance` and `market_value` attributes from
+  ALAsset will be meaningful.
+id: one vehicle info
+generic object: ALVehicle
+question: |
+  Tell us about your vehicle
+fields:
+  - Year: x.year
+    input type: combobox
+    code: |
+      recent_years()
+  - Make: x.make
+    hint: e.g, Ford, Honda
+  - Model: x.model
+    hint: e.g., Fusion, Civic
```

Comment:
I know I added these `hints` but I think we should replace with the normal `help` feature--`hint` text is disfavored: https://www.nngroup.com/articles/form-design-placeholders/